doc/users/whats_new.rst

```diff
@@ -70,80 +118,86 @@ labels will be parallel to the circular grid line, and angular tick labels will
 be perpendicular to the grid line (i.e., parallel to the outer boundary.)
 
 
-Merge JSAnimation
------------------
+``Figure`` class now has ``subplots`` method
+--------------------------------------------
 
-Jake Vanderplas' JSAnimation package has been merged into matplotlib. This
-adds to matplotlib the `~matplotlib.animation.HTMLWriter` class for
-generating a javascript HTML animation, suitable for the IPython notebook.
-This can be activated by default by setting the ``animation.html`` rc
-parameter to ``jshtml``. One can also call the
-`~matplotlib.animation.Animation.to_jshtml` method to manually convert an
-animation. This can be displayed using IPython's ``HTML`` display class::
+The :class:`~matplotlib.figure.Figure` class now has a
+:meth:`~matplotlib.figure.Figure.subplots` method which behaves the same as
+:func:`.pyplot.subplots` but on an existing figure.
 
-    from IPython.display import HTML
-    HTML(animation.to_jshtml())
 
-The `~matplotlib.animation.HTMLWriter` class can also be used to generate
-an HTML file by asking for the ``html`` writer.
+Metadata savefig keyword argument
+---------------------------------
 
+:func:`~matplotlib.pyplot.savefig` now accepts ``metadata`` as a keyword
+argument. It can be used to store key/value pairs in the image metadata.
 
+Supported formats and backends
```

Comment:
I don't think the backends are really relevant, because savefig will do the backend switch itself anyways?  The deep nesting looks a bit silly too.  I would just say

... in the image metadata (png metadata keys are ..., pdf metadata key are documented in `writeInfoDict`, and ps and eps only support the "Creator" key)

Reply:
Multiple backends can write png, if you are using a non-agg backend that can write a png it will not switch.

Reply:
Ah, true...